src/picomsg/src/oper_db_structs.h

```diff
@@ -29,3 +29,10 @@ struct terminator_tunnel_oper_data {
 	const char *local_network;
 	const char *neigh_sub_interface;
 };
+
+struct terminator_tunnel_lifetime_intervals {
```

Comment:
The writes are not for a terminator tunnel, they are for just "tunnels" (AKA configured-tunnels).
So rename to something like `struct configured_tunnel_lifetime_intervals`